templates/web/pages/user_form/user_form_page.tt.html

```diff
@@ -257,4 +255,6 @@ <h3>[% lang("you_can_also_help_us") %]</h3>
     <p>type: [% type %] action: [% action %] userid: [% userid %]</p>
 [% END %]
 
+<script type="text/javascript" src="/js/user_form.js"></script>
```

Comment:
This include is probably not necessary any more with https://github.com/openfoodfacts/openfoodfacts-server/blob/c1ef2354207a96fb31015a58bbb783cadf092841/templates/web/common/site_layout.tt.html#L417 being in place?